pom.xml

```diff
@@ -19,12 +19,12 @@
     <parent>
         <groupId>org.eclipse.microprofile</groupId>
         <artifactId>microprofile-parent</artifactId>
-        <version>2.6</version>
+        <version>3.2</version>
```

Comment:
IIRC we can't move up to the 3.2 parent because that would change dependencies to a newer version of Jakarta EE.